nixos/modules/services/backup/sanoid.nix

```diff
@@ -15,38 +15,38 @@ let
   commonOptions = {
     hourly = mkOption {
       description = "Number of hourly snapshots.";
-      type = types.ints.unsigned;
-      default = 48;
+      type = with types; nullOr ints.unsigned;
+      default = null;
```

Comment:
Good catch here. This was changed from using null defaults late in the review cycle for the original PR (https://github.com/NixOS/nixpkgs/pull/72060#issuecomment-583912497) and I didn't consider the side effects.

cc @Infinisil

Reply:
Agree, when reading the nixos docmentation, `null` can be confusing. I'd rather like having an explicit default.